static/js/narrow_state.js

```diff
@@ -154,7 +154,7 @@ export function pm_ids_string() {
     return user_ids_string;
 }
 
-export function pm_emails_string() {
+export function pm_email_string() {
```

Comment:
Oops, it's a bit unfortunate that you changed the name in this commit, not the original one. Though I'm not sure why `pm_email_string` is a better name -- can't it include more than one email address, when the draft is to a group PM?

Reply:
Oh hm I don't remember doing that -- I think `pm_emails_string` is fine. Thanks!